### ISSUE TITLE
Applications using OpenGL segfault on the Arch image if you don't install libxkbcommon

### ISSUE THREAD

**Issue by quyykk** [ruanformigoni/flatimage#30]
I'm getting a segfault running various OpenGL applications unless I install libxkbcommon. I'm using the 0.2.4 Arch image on a Arch host. I tried a couple of games and they all segfault.

However, if I manually install libxkbcommon, they stop segfaulting and work perfectly.

Reproduction steps:

```bash
$ ./arch.flatimage fim-resize 4G
$ ./arch.flatimage fim-perms-set wayland,pulseaudio,gpu,session_bus
$ ./arch.flatimage fim-root fakechroot pacman -S supertux --noconfirm
$ ./arch.flatimage fim-exec supertux2
# doesn't start, segfaults
$ ./arch.flatimage fim-root fakechroot pacman -S libxkbcommon --noconfirm
$ ./arch.flatimage fim-exec supertux2
# starts without problems
```

**Comment by ruanformigoni** [comment]
Thanks for the heads up! I'll pre-package libxkbcommon to fix this.

**Comment by ruanformigoni** [comment]
Included in v0.2.5, if the issue persists, feel free to re-open.
